src/panels/media-browser/ha-panel-media-browser.ts

```diff
@@ -69,15 +78,25 @@ class PanelMediaBrowser extends LitElement {
                     .narrow=${this.narrow}
                   ></ha-menu-button>
                 `}
-            <div main-title class="heading">
-              <div>
-                ${!this._currentItem
-                  ? this.hass.localize(
-                      "ui.components.media-browser.media-player-browser"
-                    )
-                  : this._currentItem.title}
-              </div>
+            <div main-title>
+              ${!this._currentItem
+                ? this.hass.localize(
+                    "ui.components.media-browser.media-player-browser"
+                  )
+                : this._currentItem.title}
             </div>
+            ${this._currentItem &&
+            isLocalMediaSourceContentId(
+              this._currentItem.media_content_id || ""
+            )
+              ? html`
+                  <ha-icon-button
+                    .label=${this.hass!.localize("ui.panel.map.edit_zones")}
```

Comment:
```suggestion
                    .label=${this.hass!.localize("ui.panel.map.edit_zones")}
```
We are not editing zones are we? 😄